android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1143,11 +1188,13 @@ public void setFullscreen(boolean fullscreen) {
                 uiOptions = SYSTEM_UI_FLAG_HIDE_NAVIGATION
                         | SYSTEM_UI_FLAG_FULLSCREEN;
             }
+            activity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
             eventEmitter.fullscreenWillPresent();
             decorView.setSystemUiVisibility(uiOptions);
             eventEmitter.fullscreenDidPresent();
         } else {
             uiOptions = View.SYSTEM_UI_FLAG_VISIBLE;
+            activity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
```

Comment:
One last thought:
Forcing this to protrait can break on some use cases.
Can we track the initial orientation and restore it here?
I mean, if the app is forced to landscape even not being in fullscreen video, when returning from FS will force portrait, which may not be the desired behaviour

Reply:
@CHaNGeTe - Thanks for the suggestion.

I have tracked the Initial orientation and updated to the activity in the set fullscreen method.

Please review and merge the PR.